types/Contracts/Validation/ValidationRule.php

```diff
@@ -4,7 +4,7 @@
 
 use function PHPStan\Testing\assertType;
 
-new class implements ValidationRule
+$class = new class implements ValidationRule
```

Comment:
```suggestion
new class implements ValidationRule
```

Reply:
The instance must be assigned to some variable to avoid the following errors...

```
 ------ ----------------------------------------------------------------------------------------------------------------------------- 
  Line   ValidationRule.php                                                                                                           
 ------ ----------------------------------------------------------------------------------------------------------------------------- 
  7      Expression "new class implements \Illuminate\Contracts\Validation\ValidationRule…" on a separate line does not do anything.  
         🪪  expr.resultUnused                                                                                                        
 ------ ----------------------------------------------------------------------------------------------------------------------------- 
```